Gemfile

```diff
@@ -30,16 +30,17 @@ gem 'jbuilder', '~> 2.5'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 gem 'devise'
-
+gem 'faker', git: 'https://github.com/stympy/faker.git', branch: 'master'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 gem 'devise_token_auth'
+gem 'factory_bot'
```

Comment:
Always lock the version to avoid unexpected behaviors if someone runs `bundle update`.